modules/keyvault/secret-autorotation/main.bicep

```diff
@@ -0,0 +1,348 @@
+targetScope = 'resourceGroup'
+
+// params
+@description(
+  '''
+  Required.
+
+  Definition of secrets to be auto-rotated. Includes name of the CosmosDB, name of the KeyVault, name of the Secret, name of the Resource Group etc.
+  Example:
+  [
+    {
+      type: 'cosmosdb', // mandatory, can be 'cosmosdb', 'redis'
+      resourceName: 'cosmosdb-1',
+      resourceRg: 'resource-group-1', // optional, default to parameter "resourceGroup().name"
+      keyvaultRg: 'resource-group-1', // optional, default to parameter "resourceGroup().name"
+      keyvaultName: 'keyvault-1',
+      secretName: 'secret-1',
+      isCreate: false, // optional, default to false. If true this module will create the secret with the CosmosDB key as value in the keyvault.
+      validityDays: 90, // optional, default to 90 days
+      expirationDate: '2023-05-02T15:16:13Z' // optional, default to 90 days from now
+    }
+  ]
+  It's unlikely that for a single CosmosDB both primary and secondary keys are stored as keyvault secrets, normally one is used and later alternated to the other. Thus for now multiple keys are not supported.
+  Note! Currently only support each CosmosDB having 1 secret.
+  '''
+)
```

Comment:
```suggestion
@description('''
  Required.

  Definition of secrets to be auto-rotated. Includes name of the CosmosDB, name of the KeyVault, name of the Secret, name of the Resource Group etc.
  Example:
  [
    {
      type: 'cosmosdb', // mandatory, can be 'cosmosdb', 'redis'
      resourceName: 'cosmosdb-1',
      resourceRg: 'resource-group-1', // optional, default to parameter "resourceGroup().name"
      keyvaultRg: 'resource-group-1', // optional, default to parameter "resourceGroup().name"
      keyvaultName: 'keyvault-1',
      secretName: 'secret-1',
      isCreate: false, // optional, default to false. If true this module will create the secret with the CosmosDB key as value in the keyvault.
      validityDays: 90, // optional, default to 90 days
      expirationDate: '2023-05-02T15:16:13Z' // optional, default to 90 days from now
    }
  ]
  It's unlikely that for a single CosmosDB both primary and secondary keys are stored as keyvault secrets, normally one is used and later alternated to the other. Thus for now multiple keys are not supported.
  Note! Currently only support each CosmosDB having 1 secret.
  ''')
```

Reply:
adjust where the newlines are to improve the folding.
<img width="215" alt="image" src="https://user-images.githubusercontent.com/9027725/232362730-7995ea9d-d980-4cb6-9b28-c1565553bdce.png">
compared to:
<img width="220" alt="image" src="https://user-images.githubusercontent.com/9027725/232362703-c40d08f2-35f3-464c-95b7-068c74d35106.png">